webui-src/app/theme.css

```diff
@@ -123,7 +123,9 @@ hr {
 .widget-half {
   max-width: 50%;
 }
-
+.widget-2{
+  margin-top: -12rem;
+}
```

Comment:
What is this for? Don't think it's being used anywhere

Reply:
I have used it 11 times in app.js

Reply:
But that is not related to the navbar changes for this PR, we can include this when we are working on app.js.

Reply:
I think stylings are getting applied from there only, there are other classes also like widget-half which is providing styles to the page. I needed to move things upwards that's why I have applied that class to some elements which were not in the correct position

Reply:
Oh, so you were applying this class to elements. In that case you should commit that as well.

Reply:
Do I have to commit it individually?

Reply:
You can commit them in this PR itself.

Reply:
ok